File: ffmpeg-mini/src/main/java/com/coder/ffmpeg/call/CommonCallBack.java
package com.coder.ffmpeg.call;

/**
 * @author: AnJoiner
 * @datetime: 19-12-17
 */
public abstract class CommonCallBack implements ICallBack {

    @Override
    public void onError(Throwable t) {
        t.printStackTrace();
    }

    @Override
    public void onStart() {

    }

    @Override
    public void onProgress(int progress) {

    }
}

File: ffmpeg-mini/src/main/java/com/coder/ffmpeg/call/CommonCallBack.kt
package com.coder.ffmpeg.call

/**
 * @author: AnJoiner
 * @datetime: 19-12-17
 */
open class CommonCallBack : IFFmpegCallBack {
    override fun onStart() {}
    override fun onProgress(progress: Int, pts: Long) {}
    override fun onCancel() {}
    override fun onComplete() {}
    override fun onError(errorCode: Int, errorMsg: String?) {}
}